app/src/client/components/buildPage/PropertySchemaParser.ts

```diff
@@ -219,7 +224,11 @@ export class PropertySchemaParser implements PropertySchemaParserInterface {
   }
 
   private getNonDropdownDefaultValue(key: string, property: any): any {
-    return this.activeModelObj?.json_str?.[key] ?? property.default ?? '';
+    if (this.activeModelObj) {
+      return this.activeModelObj?.json_str?.[key] ?? '';
```

Comment:
I am not sure first question mark (`this.activeModelObj?.`) is necessary because `if` now checks if t`his.activeModelObj?` is nullish.

Reply:
Yup, the first question mark is not needed in this case.